components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,862 @@
+&= \left( \nabla_z \cdot   {\bf u}  \right) \rho {\bf u} 
++ \left( {\bf u} \cdot \nabla_z  \rho  \right) {\bf u} 
++ \left( {\bf u} \cdot \nabla_z  {\bf u} \right)  \rho
++ \partial_z \left( \rho w {\bf u}\right) 
+$$ (adv2d-prod)
+
+The term ${\bf u} \cdot \nabla_z {\bf u}$ may be replaced with the vector identity
+
+$$
+\begin{aligned}
+{\bf u} \cdot \nabla_z {\bf u}
+&= (\nabla_z \times {\bf u}) \times {\bf u} + \nabla_z \frac{|{\bf u}|^2}{2} \\
+&= \left( \boldsymbol{k} \cdot (\nabla_z \times {\bf u})\right)
+\left( \boldsymbol{k} \times {\bf u} \right) + \nabla_z \frac{|{\bf u}|^2}{2} \\
+&= \zeta {\bf u}^{\perp} + \nabla_z K,
+\end{aligned}
+$$ (advection-identity)
+
+where $\zeta$ is relative vorticity and $K$ is kinetic energy. This step separates the horizontal advection into non-divergent and non-rotational components, which is useful in the final TRiSK formulation.
+
+#### Final Continuous Equations
+
+The final form of the continuous conservation equations for a non-Boussinesq, hydrostatic ocean are
+
+momentum:
+
+$$
+\frac{\partial (\rho \mathbf{u})}{\partial t}
++ \nabla \cdot (\rho \mathbf{u} \otimes \mathbf{u})
++ \partial_z (\rho \mathbf{u} w)
++ f \rho {\bf u}^\perp 
+  = -  \nabla_z p
+  - \rho\nabla_z \Phi 
++ \rho{\bf D}^u + \rho{\bf F}^u
+$$ (continuous-momentum-final)
+
+mass:
+
+$$
+\frac{\partial \rho }{\partial t}
+ + \nabla_z \cdot \left( \rho  {\bf u}  \right)
+ + \frac{\partial}{\partial z} \left(  \rho  w \right)
+= 0
+$$ (continuous-mass-final)
+
+tracers:
+
+$$
+\frac{\partial \rho \varphi}{\partial t}
+ + \nabla_z \cdot \left( \rho \varphi {\bf u}  \right)
+ + \frac{\partial}{\partial z} \left(  \rho \varphi w \right)
+= D^\varphi + Q^\varphi
+$$ (continuous-tracer-final)
+
+equation of state:
+
+$$
+\rho = f_{eos}(p,\Theta,S).
+$$ (continuous-eos-final)
+
+hydrostatic pressure:
+
+$$
+p(x,y,z) = p^{surf}(x,y) + \int_{z}^{z^{surf}} \rho g dz'.
+$$ (continuous-hydrostatic-pressure-final)
+
+Here the $\nabla_z$ operators are exactly horizontal; we expand terms for tilted layers in the following section. The momentum diffusion terms include Laplacian (del2), biharmonic (del4), and vertical viscosity,
+
+$$
+{\bf D}^u
+= \nu_2 \nabla_z^2 {\bf u} - \nu_4 \nabla_z^4 {\bf u}
+ + \frac{\partial }{\partial z} \left( \nu_v \frac{\partial {\bf u}}{\partial z} \right)
+$$ (continuous-h_mom_diff)
+and may also include a Rayleigh drag and eventually parameterizations. Momentum forcing is due to wind stress and bottom drag. Similarly, the tracer diffusion terms include Laplacian (del2), and vertical viscosity,
+
+$$
+D^\varphi =
+ \nabla_z\cdot\left(\rho \kappa_2 \nabla_z\varphi \right)
++ \rho \frac{\partial }{\partial z}
+  \left( \kappa_v \frac{\partial \varphi}{\partial z} \right),
+$$ (continuous-v_tr_diff)
+
+and may also include a biharmonic (del4) term and parameterizations such as Redi mixing. Sources and sinks include surface fluxes from the atmosphere and land, and bio-geo-chemical reactions.
```

Comment:
@mark-petersen I don't believe these equations have been reynolds averaged at this stage.  As such, I don't think it's appropriate to have all these parameterization of reynolds stresses and turbulent fluxes in the diffusion terms.  Perhaps the original equations from Kundu or Mom5 manual are already averaged?  If so we should say so.  Otherwise only molecular diffusion/viscosity should be here in these terms in my view.